pkg/server/server.go

```diff
@@ -2655,10 +2669,18 @@ func (s *BgpServer) GetBgp(ctx context.Context, r *api.GetBgpRequest) (*api.GetB
 				ListenPort:       g.Config.Port,
 				ListenAddresses:  g.Config.LocalAddressList,
 				UseMultiplePaths: g.UseMultiplePaths.Config.Enabled,
+				StartedAt: config.ProtoTimestamp(s.status.startedAt),
+				ReconfiguredAt: config.ProtoTimestamp(s.status.reconfiguredAt),
+				Version: &api.Version{
+					Major: 2,
+					Minor: 4,
+					Patch: 0,
+					Version: "2.4.0",
+				},
```

Comment:
I think that embedding static numbers in the code has two problems:
- We have multiple versions that return 2.4.0. We should use git hash or something unique for not released tagged versions.
- We have to update these numbers every time we release a new version.

We need a way to embed a unique version for every commit automatically.